core/scripts/server.js

```diff
@@ -125,7 +125,20 @@ app.get('*', (req, res, next) => {
   const s = Date.now()
   const errorHandler = err => {
     if (err && err.code === 404) {
-      res.redirect('/page-not-found')
+      const contains = (target, pattern) => {
+        var value = 0
+        pattern.forEach(word => {
+          value = value + target.includes(word)
+        })
+        return (value >= 1)
+      }
+      if (contains(req.url, config.redirect404.notAllowedExtensions)) {
+        console.error(`Resource not found : ${req.url}`)
```

Comment:
I belive we should set the proper response  http header in here as well

Reply:
Done

Reply:
I like it!